modules/ROOT/pages/mtf-tools.adoc

```diff
@@ -8,7 +8,7 @@ MTF Tools is an extension with useful operations to test your module.
 == Queue operations
 
 Allows you to manage temporary queues during your test runs.
-All queues will be *emptied as soon as the test ends*.
+All queues will be `emptied as soon as the test ends`.
```

Comment:
Remove the backticks here '`'.
We use them to note user entries' examples. 
At the moment, we don't use enriched text to emphasize statements. It's important to follow this so the translation teams knows what to localize and how.